packages/grafana-data/src/valueFormats/dateTimeFormatters.ts

```diff
@@ -313,6 +313,24 @@ export function toDurationInHoursMinutesSeconds(size: number): FormattedValue {
   return { text: strings.join(':') };
 }
 
+export function toDurationInDaysHoursMinutesSeconds(size: number): FormattedValue {
```

Comment:
I think we probably can improve this code a bit but it does follow the same style as the other ones so lets keep it as it is. Great job!